src/components/ModalConfirmClearStorage.js

```diff
@@ -0,0 +1,67 @@
+import React, { useEffect, useState, useCallback } from 'react';
+import { t } from 'ttag';
+import $ from 'jquery';
+import SpanFmt from './SpanFmt';
+import { CONFIRM_RESET_MESSAGE } from '../constants';
```

Comment:
Do you have this?

Reply:
This is the same constant used by the wallet reset modal.
"I want to reset my wallet"